command/agent/command.go

```diff
@@ -406,7 +406,15 @@ func (c *Command) setupLoggers(config *Config) (*GatedWriter, *logWriter, io.Wri
 
 // setupAgent is used to start the agent and various interfaces
 func (c *Command) setupAgent(config *Config, logOutput io.Writer, logWriter *logWriter) error {
-	c.Ui.Output("Starting Consul agent...")
+	var version string
+
+	version = "v" + config.Version
+
+	if len(config.VersionPrerelease) != 0 {
+		version += " " + config.VersionPrerelease
+	}
+
```

Comment:
Can you please also add the revision string so the full version is formatted the same as https://github.com/hashicorp/vault/blob/master/version/version.go#L44-L57.